### PR TITLE
New: added priyankar pal

### PR DESCRIPTION
**Pull request by priyankarpal**
* Added my name.
* removed $ sign 

image

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please revert the other changes and keep your profile additions only.
Let us know if you need any help with doing that. Thanks. 🙂

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by mahfuztbt — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by SurajPratap10 — APPROVED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts [**here**](https://github.com/Design-and-Code/Opensource-practice/pull/48/conflicts). If you need any help, you can ask for help here or on our [**Discord server**](https://discord.gg/gM3bG4rAU5). You can also check the [GitHub Docs](https://docs.github.com/en/github/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github) to make the process easier. 🙂

**Review by SurajPratap10 — APPROVED** [review]
looks good now

File: README.md [hunk 1]
```diff
@@ -198,3 +199,4 @@ git push -u origin <your_branch_name>
 <div align="center">
 
 We will be happy if you give this repository a ⭐
+
```
Comment (Panquesito7) [inline README.md:202]:
```suggestion
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**